StreamChatSwiftUITestsAppTests/Robots/UserRobot.swift

```diff
@@ -336,13 +395,14 @@ extension UserRobot {
 extension UserRobot {
 
 //    @discardableResult
-//    func tapOnDebugMenu() -> Self {
```

Comment:
These are commented out because they're missing some features in the test app? We can uncomment them if that's the case.

Reply:
Done 👍